docs/core/seeding.rst

```diff
@@ -288,19 +358,20 @@ This function assigns a weight (which should correspond to the likelihood that
 a seed is good) to all seeds and calls the detector specific cuts to apply a
```

Comment:
calls -> applies detector specific section of seeds based on weights. (not clear what the weight modifications is for)